docs/validator/running-a-validator.md

```diff
@@ -0,0 +1,252 @@
+---
+id: running-a-validator
+---
+
+# Running a Validator
```

Comment:
This tutorial should be split up into multiple ones, for different use cases:
- fresh chain
- connection to an existing chain
- setting up node secrets / node data dir

Reply:
We already have 2 sections.

1. `# Start a New Gno Chain and a Validator`
- for fresh chain(+ and optional to set validator priv key, state, and nodekey)

2. `# Connect to an Existing Gno Chain`

Reply:
1731db0b3c344a7e76b53577c5302bdef552d111